cli/ballerina-packerina/src/test/java/org/ballerinalang/packerina/cmd/NewCommandTest.java

```diff
@@ -36,34 +36,88 @@
 
     @Test(description = "Initialize a new empty project")
     public void testNewCommand() throws IOException {
-        String[] args = {"project-name"};
+        String[] args = {"project_name"};
         NewCommand newCommand = new NewCommand(tmpDir, printStream);
         new CommandLine(newCommand).parse(args);
         newCommand.execute();
         // Check with spec
-        // project-name/
+        // project_name/
         // - Ballerina.toml
-        // - src/
-        // - tests/
-        // -- *.bal           <- integration test code
-        // -- resources/      <- integration test resources
+        // - Package.md
+        // - Module.md
+        // - main.bal
+        // - resources
+        // - tests
+        //      - main_test.bal
+        //      - resources/
         // - .gitignore       <- git ignore file
 
-        Path projectPath = tmpDir.resolve("project-name");
-        Assert.assertTrue(Files.exists(projectPath));
-        Assert.assertTrue(Files.exists(projectPath.resolve("Ballerina.toml")));
-        Assert.assertTrue(Files.exists(projectPath.resolve("src")));
-        Assert.assertTrue(Files.isDirectory(projectPath.resolve("src")));
-        //Path integrationTestPath = projectPath.resolve("tests");
-        //Assert.assertTrue(Files.exists(integrationTestPath));
-        //Assert.assertTrue(Files.isDirectory(integrationTestPath));
-        //Assert.assertTrue(Files.exists(integrationTestPath.resolve("resources")));
-        //Assert.assertTrue(Files.isDirectory(integrationTestPath.resolve("resources")));
-        Assert.assertTrue(Files.exists(projectPath.resolve(".gitignore")));
+        Path packageDir = tmpDir.resolve("project_name");
+        Assert.assertTrue(Files.exists(packageDir));
+        Assert.assertTrue(Files.exists(packageDir.resolve(ProjectConstants.BALLERINA_TOML)));
```

Comment:
New Command without any args should default to the "main" template. Therefore, we should add an assertion to verify that the `main.bal` file exists.